query/src/provider/overlap.rs

```diff
@@ -179,15 +180,16 @@ where
     /// there were no NULLs
     fn potential_overlap(&self, other: &Self) -> Result<bool> {
         // This algorithm assumes that the keys are sorted by name (so
-        // they can't appear in different orders on the two sides)
+        // they can't appear in different orders on the two sides) except
+        // the "time"column which is always the last column
```

Comment:
Looks like there should be a space in here?

```suggestion
        // the "time" column which is always the last column
```